cmake/Platform/Linux/Packaging/snapcraft.yaml.in

```diff
@@ -0,0 +1,64 @@
+name: o3de
+version: '${LY_VERSION_STRING}'
+summary: O3DE Engine
+description: |
+  O3DE Engine
+confinement: classic
+base: core22
+
+parts:
+  test-o3de:
+    plugin: dump
+    source: ./O3DE
```

Comment:
is uppercase O3DE as a foldername safe? when you clone does it not usually use lowercase or is this for the name where the source is installed?

Reply:
This is using the generated install layout. which uses an uppercase O3DE. This does come from CPACK_PACKAGE_NAME so maybe the better option here is to use that variable in the snapcraft file.